docs/integrations/builtin/core-nodes/n8n-nodes-langchain.mcptrigger.md

```diff
@@ -94,7 +94,16 @@ Be sure to replace the `<MCP_URL>` and `<MCP_BEARER_TOKEN>` placeholders with th
 
 ## Limitations
 
-The MCP Server Trigger node doesn't yet support n8n instances running in [queue mode](/hosting/scaling/queue-mode.md) mode.
+**MCP Trigger and Webhook Replicas**  
+The MCP Trigger node relies on Server-Sent Events (SSE), which require the same server instance to handle persistent connections.
```

Comment:
```suggestion
The MCP Server Trigger node relies on Server-Sent Events (SSE), which require the same server instance to handle persistent connections. This can cause problems when running n8n in [queue mode](/hosting/scaling/queue-mode.md) depending on your [webhook processor](/hosting/scaling/queue-mode.md#webhook-processors) configuration:
```